int16 stepper_number
bool stepper_direction
int16 stepper_count
